TrackerExtenstion.cs

```diff
@@ -38,14 +39,48 @@ public void Startup(StartupParams sp)
         /// </param>
         public void Ready(ReadyParams rp)
         {
+            // @todo Subscribe to an event after the Dynamo home screen is shown
+
+            // Display a MessageBox to the user
+            // @todo Use a view framework to improve the UI/UX
             string message = "By pressing OK you agreeing to Binoculars 🔍 data collection.";
             string title = "Terms of Use Agreement";
-
             MessageBox.Show(message, title, MessageBoxButton.OK, MessageBoxImage.Exclamation);
+
+            // @todo if user does not consent, don't store..
+            if (true) {
+
+                // Dynamo StartupParams
+                ExportData.dynamoVersion = rp.StartupParams.DynamoVersion.ToString();
+
+                // @todo Determine whether the script is run from DynamoPlayer..
+
+                // Store the local environment data
+                ExportData.user = Environment.MachineName;
+                ExportData.computerName = Environment.UserName;
+                
+                // @todo Define the revitVersion, leave blank or null if opened from any other environment (Sandbox, Civil3D etc)
+                ExportData.revitVersion = "2019.0.2";
```

Comment:
use the Dynamo host information for this.

Reply:
New issue created: https://github.com/teamtreedyn/Binoculars/issues/21 therefore I'm marking resolved.